CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# How to Contribute
+
+Contributions are essential for keeping this language server great. We try to keep it as easy as possible to contribute changes and we are open to suggestions for making it even easier. There are only a few guidelines that we need contributors to follow.
+
+## Development
+
+### Installation Prerequisites:
+
+  * [JDK 8+](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```

Comment:
Maybe link to AdoptOpenJDK instead? It looks like it's [joining the Eclipse foundation](https://blog.adoptopenjdk.net/2020/06/adoptopenjdk-to-join-the-eclipse-foundation/)

Reply:
Done. We may also want to change this on the vscode-xml page.